src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx

```diff
@@ -13,20 +13,20 @@ redirects:
   - /docs/chart-builder/use-chart-builder/use-charts/use-your-charts
 ---
 
-Once created, you can customize the appearance of your chart to best present the data. You can share a chart in different formats and add a chart to a new or existing dashboard.
+Once you've created a chart, you can customize the appearance of it to best present the data. You can share a chart in different formats and add a chart to a new or existing dashboard.
 
 ## Change the appearance of your chart [#change-appearance]
 
 When you run your query in **advanced (NRQL) mode** or view your chart while using **basic mode** to specify data, the query builder analyzes your data and applies a [chart type](/docs/insights/use-insights-ui/manage-dashboards/insights-chart-types) that fits your data.
 
-You can select another chart type that you prefer — your chart should tell the story that you want to communicate. Some chart types may not be available due to the nature of the data you specified.
+For some queries, you'll have several options of chart types to choose from. You should attempt to choose the type that best fits your needs. 
```

Comment:
The second sentence, "attempt to choose the type that best fits..." is kind of non-information. Is there anything more substantial to be said? If not, maybe just delete that second sentence.